kernel-laws/src/main/scala/cats/kernel/laws/MonoidLaws.scala

```diff
@@ -0,0 +1,28 @@
+package cats.kernel.laws
+
+import cats.kernel.Monoid
+
+trait MonoidLaws[A] extends SemigroupLaws[A] {
```

Comment:
I couldn't find the replacement for `Rules.isId("isEmpty", A.empty)(A.isEmpty)`?